docs/mine/lotus/seal-workers.md

```diff
@@ -144,15 +152,28 @@ export LOTUS_WORKER_PATH=/path/to/worker/storage/N
 lotus-worker run --listen 0.0.0.0:X --add-piece=false --precommit1=true --unseal=true --precommit2=false --commit=false
 ```
 
-Next, it is important to make sure each worker is limited to use a **single CPU**. This will limit the available tasks that each _PreCommit1_ workers can perform to 1. This limit can be enforced by creating systemd services for every worker and [setting _CPU affinity_ with systemd](https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html):
+By default, the _PreCommit1_ base will _use a single CPU core_. This means that several workers process can use remaining cores and perform the work in parallel for this phase, as long as there is enough memory available for all of them to run.
+
+Alternatively, it is also possible to speed up _PreCommit1_ by setting `FIL_PROOFS_USE_MULTICORE_SDR=1`. This enables [optimizations to memory access](https://github.com/filecoin-project/rust-fil-proofs/), but each worker will need access to a full CPU core-complex (usually 4 adjacent cores). This reduces the number of workers that could run in parallel, but allows them to be faster and saves the memory footprint of running additional processes.
+
+In both cases, whether using the single-core or the multicore _PreCommit1_ worker, the process should be restricted to the use of a **single CPU** or a **single, full core-complex** respectivly. This can be achieved with `taskset`:
+
+```sh
+# Restrict to single core number 0
+taskset -c 0 <worker_pid | command>
+# Restrict to a single core complex (example)
+# Check your CPU model documentation to verify how many
+# core complexes it has and how many cores in each:
+taskset -c 0,1,2,3 <worker_pid | command>
+```
+
+It is also possible to set [_CPU affinity_ with systemd](https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html):
```

Comment:
This will be done internally in the proofs library, in a way where a single process can run multiple parallel precommits assigned to different cores, users shouldn't need to do this when we get this into lotus.

Running multiple workers on a single machine is currently not supported and will result in things running out of memory

Reply:
> This will be done internally

In the future, or currently happening?

> Running multiple workers on a single machine is currently not supported and will result in things running out of memory

We recommended the opposite, https://filecoin.io/vintage/mining-hardware-config-testnet-v3.pdf . @lanzafame also mentioned this strategy I think.

Reply:
> In the future, or currently happening?

I remember @porcuquine said this is a future feature (and `taskset` tip comes from lotus-proofs anyways).

In terms of memory, it is mentioned that parallel workers take more memory.